<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>265</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Agency FB</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="xd">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Symbol</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AÑO ACTUAL:</string>
    </property>
   </widget>
   <widget class="QLabel" name="xf">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Symbol</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AÑO DE NACIMIENTO:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtN1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtN2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="xs">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Symbol</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SU EDAD ES:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtN3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCalculo">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>89</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CALCULAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>89</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LIMPIAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>89</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>582</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
